api/src/pcapi/models/feature.py

```diff
@@ -128,6 +128,7 @@ class FeatureToggle(enum.Enum):
     WIP_ENABLE_NEW_COLLECTIVE_OFFERS_AND_BOOKINGS_STRUCTURE = (
         "Activer la nouvelle structure des offres et réservations collectives"
     )
+    WIP_ALGOLIA_SEARCH_BO = "Activer la recherche algolia dans le BO"
```

Comment:
Suggestion : En général on met « enable » dans le nom du FF.
```suggestion
    WIP_ENABLE_ALGOLIA_SEARCH_IN_BO = "Activer la recherche algolia dans le BO"
```